README.md

```diff
@@ -109,6 +109,25 @@ const config: PlaywrightTestConfig = {
 export default config
 ```
 
+#### Report options
+
+You can configure options that should be passed to the aXe HTML reporter at
+the assertion level.
+
+```js
+await expect(someLocator).toBeAccessible({
+  reportFileName: 'my-report.html',
+})
+
+await expect(someOtherLocator).toBeAccessible({
+  reportFileName: 'my-other-report.html',
+})
```

Comment:
Forgot we need to update to `filename` in the readme, also we can simplify the example a bit.

```suggestion
await expect(page.locator('#my-element')).toBeAccessible({
  filename: 'my-report.html',
})
```

Reply:
🤦‍♂️ My bad, will make that change now